packages/expo-facebook/ios/EXFacebook/EXFacebook.m

```diff
@@ -70,13 +76,30 @@ @implementation EXFacebook
   resolve(nil);
 }
 
+UM_EXPORT_METHOD_AS(logOutAsync,
+                    logOutAsync:(UMPromiseResolveBlock)resolve
+                    rejecter:(UMPromiseRejectBlock)reject)
+{
+  FBSDKLoginManager *loginManager = [[FBSDKLoginManager alloc] init];
+  [loginManager logOut];
+  resolve(nil);
+}
+
+UM_EXPORT_METHOD_AS(getAccessTokenAsync,
+                    getAccessTokenAsync:(UMPromiseResolveBlock)resolve
+                    rejecter:(UMPromiseRejectBlock)reject)
+{
+  FBSDKAccessToken *currentAccessToken = [FBSDKAccessToken currentAccessToken];
+  resolve(UMNullIfNil([EXFacebook accessTokenNativeToJSON:currentAccessToken]));
+}
+
 UM_EXPORT_METHOD_AS(logInWithReadPermissionsAsync,
                     logInWithReadPermissionsWithConfig:(NSDictionary *)config
                     resolver:(UMPromiseResolveBlock)resolve
                     rejecter:(UMPromiseRejectBlock)reject)
 {
   if (![FBSDKSettings appID]) {
-    reject(@"E_CONF_ERROR", @"No FacebookAppId configured, required for initialization. Please ensure that you're either providing `appId` to `initializeAsync` as an argument or inside Info.plist.", nil);
+    reject(EXFacebookMisconfiguredErrorDomain, @"No appId configured, required for initialization. Please ensure that you're either providing `appId` to `initializeAsync` as an argument or inside Info.plist.", nil);
     return;
   }
```

Comment:
Do you think it makes sense to extract this message string to a shared variable? Or whole `if`?